### PR TITLE
Define a skeleton section for bounce tracking spec work.

### PR DESCRIPTION
**Pull request by wanderview**
@jyasskin PTAL.  Let me know if you would like a different structure or disclaimer.  Also, not sure if I should include the doctoc update to README.md or not.

Thanks.

***
<a href="https://pr-preview.s3.amazonaws.com/wanderview/nav-tracking-mitigations/pull/34.html#bounce-tracking-mitigations" title="Last updated on Mar 29, 2023, 3:10 PM UTC (5b1ae23)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/privacycg/nav-tracking-mitigations/34/91d7936...wanderview:5b1ae23.html" title="Last updated on Mar 29, 2023, 3:10 PM UTC (5b1ae23)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -275,6 +275,33 @@ existing open-source projects.
...
+<h3 id="bounce-tracking-mitigations">Bounce Tracking Mitigations</h3>
+
+This section will provide a "monkey patch" specification for bounce tracking
+mitigations.  This work has been proposed in an
+<a href="https://github.com/privacycg/nav-tracking-mitigations/blob/main/bounce-tracking-explainer.md">explainer</a>
+but is not complete yet.  As the algorithm is developed it will be specified
+here and presented for review.
+
+<p class=note>
+The following is a work-in-progress and does not yet reflect any consensus in
```
Comment (jyasskin) [inline index.bs:287]:
```suggestion
The content of this section is a work-in-progress and does not yet reflect any consensus in
```

File: index.bs [hunk 2]
```diff
@@ -275,6 +275,33 @@ existing open-source projects.
 
 </section>
 
+<h3 id="bounce-tracking-mitigations">Bounce Tracking Mitigations</h3>
```
[the remark this answers is not in the document]
Comment (pes10k) [reply index.bs:278]: sounds good to me

**Comment by pes10k** [comment]
My main question here is why https://github.com/privacycg/nav-tracking-mitigations/blob/main/bounce-tracking-explainer.md is being added as a 5.2 level general section, instead of a "chrome" sub-subsection in 5.1.

I'm not sure I have a strong opinion either way, but I'd like to better understand the thinking behind this change

**Comment by wanderview** [comment]
> My main question here is why https://github.com/privacycg/nav-tracking-mitigations/blob/main/bounce-tracking-explainer.md is being added as a 5.2 level general section, instead of a "chrome" sub-subsection in 5.1.

From my perspective I want to write a specification and try to reach consensus with other browsers to align on it.  We don't want this to be a chrome-only approach.

From that perspective I'd like to have a non-chrome section describing the proposed behavior.  Then in the chrome section we can say "Chrome started shipping the bounce tracking mitigations described below in version X".

**Comment by wanderview** [comment]
Since you said you don't have a strong opinion and Jeffrey approved I'm going to go ahead and merge this.  We can change it of course.

**Comment by pes10k** [comment]
I see, thank you for the explanation @wanderview.

A few further points though:

One, why not bring the document into the main document, instead of having it be separate documents? Especially since i. the current Nav-tracking document already defines what bounce tracking is, ii. we define bounce-tracking as a subset of nav-tracking, and iii. and your bounce tracking doc refers back to the bounce tracking mitigations described in the nav-tracking document.

Unless there is a strong reason to split things up like this, I'd prefer to just have one document, rather than two overlapping ones.  WDYT?

And two, the bounce tracking document does seem at least semi-browser specific, and so i think it needs to be marked as such for now, or not linked to from the parent doc. 

For example, "This proposal largely only adds value when third-party cookies are disabled" is only true in Chrome and Edge currently, since FF, Safari and Brave currently enable but partition 3p cookies.  Similarly, text like "Therefore we do not plan to integrate first-party sets with our bounce tracking mitigations." sure seems very Chrome-team specific, and there are some conclusions in the document that (as best i can tell) are only the result of Chrome team discussions (it rejects what Brave currently ships, for example).

If the goal here is to have the bounce-tracking document / subsection be describing a general solution, i think it'd be best i. to mark it as currently being a chrome proposal, though one looking for broader feedback, and ii. have some discussion around it before linking to it from the parent doc (not only does it "not yet reflect any consensus in the PrivacyCG", it doesn't even have consensus of the editors of the Nav-tracking document!)

All that said, im excited you all are working in the space, and eager to combine all this into a stronger set of proposals around nav-tracking!

**Comment by jyasskin** [comment]
If you look at https://pr-preview.s3.amazonaws.com/wanderview/nav-tracking-mitigations/pull/37.html#bounce-tracking-mitigations (the preview of what's proposed in #37), you can see that @wanderview is merging the bounce tracking ideas into the main document, and that the merge is trying not to be browser-specific. I think it'd be fine to mark the link to [bounce-tracking-explainer.md](https://github.com/privacycg/nav-tracking-mitigations/blob/main/bounce-tracking-explainer.md) as Chrome-specific ... maybe as "There is a [Chromium-oriented explainer](https://github.com/privacycg/nav-tracking-mitigations/blob/main/bounce-tracking-explainer.md) for this work, but the text in this section is intended for adoption across all browsers."?

@wanderview sketched this plan in the "Navigation Tracking Status" email thread from March 7. I think it'll be easier to manage the spec as a section of this document, with reviews welcome both during PRs and after them, than it would be to write a separate specification document and have folks review the idea of merging that in. We could put this new section in a Chrome subsection of the ["deployed mitigations"](https://privacycg.github.io/nav-tracking-mitigations/#deployed-mitigations) section, but 1) I'm not sure this is all deployed yet, especially if folks suggest changes during PR review, and 2) I don't think the other browsers have written specifications of what they're doing or are pursuing interoperability around their mitigations. The pursuit of interoperability argues for making this a top-level section.

**Comment by pes10k** [comment]
> you can see that @wanderview is merging the bounce tracking ideas into the main document

Ah, i see, I just saw https://github.com/privacycg/nav-tracking-mitigations/pull/34/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R278 and thought the plan was to maintain two diff documents.  My error, sounds good!

> There is a [Chromium-oriented explainer](https://github.com/privacycg/nav-tracking-mitigations/blob/main/bounce-tracking-explainer.md) for this work, but the text in this section is intended for adoption across all browsers."?

Sounds good, that addresses my concerns.  I appreciate that its a starting place and intended to get to a cross-browser / general place, but marking it as Chrome oriented or similar in the interim would be great. I appreciate it.

> The pursuit of interoperability argues for making this a top-level section.

Totally agreed, no argument from me.  I was reacting to the text change in this PR, and didn't think / remember to consider it in the context of the previous email. Again, my error, thank you for the correction.

**Pull request merged**